src/pages/_document.js

```diff
@@ -9,12 +9,25 @@ export default class MyDocument extends Document {
     return (
       <Html lang="en">
         <Head>
+          <link
+            rel="apple-touch-icon"
+            sizes="180x180"
+            href="/apple-touch-icon.png"
+          />
+          <link
+            rel="icon"
+            type="image/png"
+            sizes="32x32"
+            href="/favicon-32x32.png"
+          />
```

Comment:
I think we can use [`Next SEO`](https://github.com/garmeeh/next-seo#additional-link-tags) for these.